lib/iris/tests/unit/fileformats/grib/test_GribWrapper.py

```diff
@@ -88,90 +88,48 @@ def test_edition_1(self):
         self.assertEqual(wrapper.grib_message, grib_message)
 
 
-@tests.skip_biggus
-class Test_deferred(tests.IrisTest):
+@tests.skip_data
+class Test_deferred_data(tests.IrisTest):
+    def test_regular_data(self):
+        filename = tests.get_data_path(('GRIB', 'gaussian',
+                                        'regular_gg.grib1'))
+        messages = list(grib._load_generate(filename))
+        self.assertTrue(is_lazy_data(messages[0]._data))
+
+    def test_reduced_data(self):
+        filename = tests.get_data_path(('GRIB', 'reduced',
+                                        'reduced_ll.grib1'))
+        messages = list(grib._load_generate(filename))
+        self.assertTrue(is_lazy_data(messages[0]._data))
+
+
+class Test_deferred_proxy_args(tests.IrisTest):
     def setUp(self):
-        confirm_patch = mock.patch(
+        confirm_patch = self.patch(
             'iris.fileformats.grib.GribWrapper._confirm_in_scope')
-        compute_patch = mock.patch(
+        compute_patch = self.patch(
             'iris.fileformats.grib.GribWrapper._compute_extra_keys')
-        long_patch = mock.patch('gribapi.grib_get_long', _mock_grib_get_long)
-        string_patch = mock.patch('gribapi.grib_get_string',
+        long_patch = self.patch('gribapi.grib_get_long', _mock_grib_get_long)
+        string_patch = self.patch('gribapi.grib_get_string',
                                   _mock_grib_get_string)
-        native_patch = mock.patch('gribapi.grib_get_native_type',
+        native_patch = self.patch('gribapi.grib_get_native_type',
```

Comment:
So you mean I could just start all those patches with 'self.patch('gribapi.stuff....')'?